CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing
 
-- Your favourite command isn't covered?
+- Your favorite command isn't covered?
```

Comment:
I don't mind strongly either way. But

> Besides, with my spelling checker set to US English I can see words with British spelling underlined in red, but not those with US spelling.

isn't a strong argument, because I have the opposite highlighting :) That being said happy to move everything to US spellings if we think that's more common.